001-IntroToKubernetes/Student/07-updaterollback.md

```diff
@@ -10,28 +10,33 @@ It's time to update your application, what do you do in a containerized world? K
 
 In this challenge you'll be deploying a v2 of the FabMedical application to your Kubernetes cluster. We're going to do this using two different update strategies: "Rolling Update" and "Blue/Green Deployment".
 
+### Update the app and load data
 - We have staged an updated version of the app on Docker Hub with id and version:
 	- **whatthehackmsft/content-web:v2**
 	- **whatthehackmsft/content-api:v2**
 - **NOTE:** If you have been building your docker container images from source code and deploying to an Azure Container Registry, you can find v2 of the source code in your Challenge 7 Resources folder.
-- Version 2 of FabMedical stores its data in MongoDB.  We have provided a container image with an initialization script called “content-init” that loads the database with the sample content (Speakers & Sessions data).  The container runs as a Kubernetes Job.  The container image is available on Dockerhub at:
-	- **whatthehackmsft/content-init**
+- Version 2 of FabMedical stores its data in MongoDB.  We have provided a container image with an initialization script called “content-init” that loads the database with the sample content (Student/Resources/Challenge 7).  The container runs as a Kubernetes Job.  The container image is available on Dockerhub at: **whatthehackmsft/content-init**
 	- Use the content-init “Job” yaml provided to run the initialization of MongoDB for our new version of the app.
-	- You should verify that the MongoDB contains the FabMedical data after content-init job has completed.
-	- Logs for content-init will provide the detailed logs showing whether it was able to successfully connect and add the contents to the MongoDB. You can use kubectl to check the logs.
-- Perform a rolling update of content-web on your cluster to the new version of content-web
-	- You’ll be doing this from the command-line with a kubectl command (remember, Kubernetes docs are your friend!)
-	- With kubectl and its watch feature you should be able to see new pods with the new version come online and the old pods terminate.
+	- You should verify that the MongoDB contains the FabMedical data after content-init job has completed.  Hint:
+    	- Connect to the mongodb pod
+    	- Use the mongodb command `show dbs`
+	- Logs for content-init will provide the detailed logs showing whether it was able to successfully connect and add the contents to the MongoDB. You can use kubectl (or the Azure Portal) to check the logs.
+
+### Rolling update
+- Next, you will need to perform a rolling update of content-web on your cluster to the new version of content-web.  You will need to edit your deployment to incorporate the following:
+  1. You need to update the container image referenced in your deployment as specified above.
+  2. The new version of content-api will need to know how to reach the MongoDB server. You will need to pass it an environment variable named: **MONGODB_CONNECTION**, and this needs to be set the URL:  `mongodb://mongodb:27017/contentdb`
```

Comment:
Please remove lines 27 & 28 , we don't want to mention content-api here because we ONLY want content-web to get a rolling update. Blue Green will update api.

Reply:
I reverted my changes per our discussion